src/styles/utils/_loops.scss

```
@@ -0,0 +1,36 @@
+$week-days :
+  "mon" ($marginsForWeekDays) * 0,
+  "tue" ($marginsForWeekDays) * 1,
+  "wed" ($marginsForWeekDays) * 2,
+  "thu" ($marginsForWeekDays) * 3,
+  "fri" ($marginsForWeekDays) * 4,
+  "sat" ($marginsForWeekDays) * 5,
+  "sun" ($marginsForWeekDays) * 6;
+
+$days: 
+  "mon" 1, 
+  "tue" 2, 
+  "wed" 3, 
+  "thu" 4, 
+  "fri" 5, 
+  "sat" 6, 
+  "sun" 7;
+
+@for $index from 28 through 31 {
+  .calendar--month-length--#{$index}
+  .calendar__day:nth-child(n + #{$index + 1}) {
+    display: none;
+  }
+}
+
+@each $day, $padding in $week-days {
+  .calendar--start-day--#{$day} .calendar__day:first-child {
+    margin-left: $padding;
```

it'd be more concise to use `$marginsForWeekDays` here instead of repeating it 7 times above
```suggestion
    margin-left: $padding * $marginsForWeekDays;
```